tlsn/tlsn-core/src/lib.rs

```diff
@@ -37,6 +37,9 @@ pub use transcript::{Direction, Transcript, TranscriptSlice};
 /// This value must not exceed bcs's MAX_SEQUENCE_LENGTH limit (which is (1 << 31) - 1 by default)
 const MAX_TOTAL_COMMITTED_DATA: u64 = 1_000_000_000;
 
+/// The encoding id
+///
+/// A 64 bit Blake3 hash which is used for the garbling labels
```

Comment:
We agreed to avoid as much as possible any GC terminology in the `tlsn` crate. We need to re-word  "labels" and "garbling" with "plaintext encodings".